package.json

```diff
@@ -8,11 +8,14 @@
     "lint": "git diff HEAD --name-only --diff-filter=ACM | grep '.js$' | xargs node ./node_modules/eslint/bin/eslint.js --quiet",
     "lint-full": "node ./node_modules/eslint/bin/eslint.js .",
     "test": "jest --config ./config/jest/config.js",
+    "test:runner": "cypress open --project config/cypress --env name=runner",
+    "test:cy-headless-screenshots": "cypress run --project config/cypress --env name=headless",
+    "test:screenshots": "PORT=8001 start-test docs 8001 test:cy-headless-screenshots",
     "webpack-analyze": "mkdir -p dist; NODE_ENV=production webpack --config config/webpack/build.js --profile --json > dist/stats.json; webpack-bundle-analyzer dist/stats.json"
   },
   "husky": {
     "hooks": {
-      "pre-commit": "npm run lint && npm audit",
+      "pre-commit": "npm run lint && npm audit && npm run test",
```

Comment:
This won't actually generate new screenshots, since the `test` script only runs the jest tests (and not the cypress stuff). Should be `npm run test:screenshots`.